Example/Backpack/Utils/FeatureStories/GroupsViewController.swift

```diff
@@ -28,7 +28,15 @@ class GroupsViewController: UITableViewController {
         super.init(style: .grouped)
         tableView.register(UITableViewCell.self, forCellReuseIdentifier: cellIdentifier)
     }
-    
+
+//    override func viewWillAppear(_ animated: Bool) {
+//        super.viewWillAppear(animated)
+//
+//        // SONIC-1385: remove testing selection
+//        let ratingIndexPath = IndexPath(row: 18, section: 1)
+//        tableView(tableView, didSelectRowAt: ratingIndexPath)
+//    }
+
```

Comment:
Can we remove this code?

Reply:
sorry about that. This is a mistake

Reply:
Fixed d32bfc3e4eb8534ccfa10c0bccb177d400efeb39